.github/workflows/build.yml

```diff
@@ -273,6 +273,12 @@ jobs:
             composer config platform --unset && composer update
             bin/composer require --dev phpunit/phpunit:^7.5 --with-all-dependencies
             ../../../phpstan.phar analyse -c ../composer.neon -l 5 src tests
+          - |
+            git clone https://github.com/pmmp/PocketMine-MP.git e2e/integration/repo
```

Comment:
this should be faster and avoid the need for checkout later
`git clone --depth 1 --branch 3.14.2 https://github.com/pmmp/PocketMine-MP.git e2e/integration/repo`

Reply:
Thank you. Unfortunately it doesn't work with commits:

```
$ git clone --depth 1 --branch 22c4c2e1e02497d0e280b4360cd49f296e3da873 https://github.com/phpmyadmin/phpmyadmin.git e2e/integration/repo
Cloning into 'e2e/integration/repo'...
warning: Could not find remote branch 22c4c2e1e02497d0e280b4360cd49f296e3da873 to clone.
fatal: Remote branch 22c4c2e1e02497d0e280b4360cd49f296e3da873 not found in upstream origin
```